docs/conf.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'CATChem'
+copyright = u'2023, NOAA'
+author = u''
+
+# The short X.Y version
+version = u''
+# The full version, including alpha/beta/rc tags
+release = u''
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autosectionlabel',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.extlinks',
+    'sphinx_design',
+    'sphinx_click',
+    'sphinx_togglebutton',
+]
+
+extlinks = {
+    'issue': ('https://github.com/ufs-community/CATChem/issues/%s', 'GH'),
+    'pull': ('https://github.com/ufs-community/CATChem/pull/%s', 'PR'),
+}
+
+autosummary_generate = True  # default in Sphinx v4
+
+autodoc_default_options = {
+    "members": True,
+    "special-members": "__init__",
+    # "undoc-members": True,
+}
+autodoc_member_order = "groupwise"
+
+napoleon_google_docstring = False
+napoleon_numpy_docstring = True
+napoleon_use_param = False
+napoleon_use_rtype = False
+napoleon_use_ivar = False  # True
+napoleon_preprocess_types = True
+
+nb_execution_timeout = 300  # in seconds, for each notebook cell (default: 30)
+# nb_execution_mode = "auto"  # don't execute if all cells have output (default)
+# nb_execution_mode = "cache"  # to speed build when working on other things
+nb_execution_mode = "off"
+nb_execution_excludepatterns = [
+]
+nb_execution_show_tb = True
+
+myst_enable_extensions = ["colon_fence"]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
```

Comment:
```suggestion
language = "en"
```
Sphinx 6 warns about `None` setting

Reply:
```
WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid language code. Falling back to 'en' (English).
```

Reply:
This was my fault :/

Reply:
@colin-harkins we might want to fix this in the future if we want to e.g. turn warnings into errors.